CHANGELOG.md

```diff
@@ -28,6 +22,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Remove frontend & promote backend [#5602](https://github.com/raster-foundry/raster-foundry/pull/5602)
 - Added additional locking for auto task unlocker [#5604](https://github.com/raster-foundry/raster-foundry/pull/5604/files)
 
+## [1.66.0] - 2021-07-27
+### Fixed
+- Corrected strategy for acquiring note text when regressing task statuses to "flagged" [#5606](https://github.com/raster-foundry/raster-foundry/pull/5606)
+
+### Added
+- Failed task lock expirations now send errors to rollbar [#5606](https://github.com/raster-foundry/raster-foundry/pull/5606)
+- Added a static page [#5607](https://github.com/raster-foundry/raster-foundry/pull/5607)
+
```

Comment:
I'm very confused about this but maybe it's fine? Maybe there was a non-`chan` changelog rotation that my `chan` just edited?